lib/vquic/ngtcp2.c

```diff
@@ -42,6 +42,8 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
+#define ENABLE_SSLKEYLOGFILE
```

Comment:
Why this?

Reply:
MIrrored from openssl. I think I did that in openssl so it could be disabled for possible compliance issues? I really don't remember. I think it would be better to wrap in #ifndef DISABLE_SSLKEYLOGFILE , but if you prefer I'll remove it entirely.

Reply:
I would rather imagine that it'd be within `CURL_DISABLE_SSLKEYLOGFILE` as then we could offer a disable option for configure in "normal" style.

I don't think it is needed for compliance since QUIC will only work with OpenSSL versions that also have the keylog support.